src/utils/daos_hdlr.c

```diff
@@ -1490,10 +1490,8 @@ dm_connect(struct cmd_args_s *ap,
 				dattr.da_props = props;
 			rc = duns_create_path(ca->dst_poh, path, &dattr);
 			if (rc != 0) {
-				rc = daos_errno2der(rc);
-				DH_PERROR_DER(ap, rc, "provide a destination pool or UNS path "
-					      "of the form:\n\t --dst </$pool> | </path/to/uns>");
-				D_GOTO(err, rc);
+				DH_PERROR_SYS(ap, rc, "failed to create destination UNS path");
+				D_GOTO(err, rc = daos_errno2der(rc));
```

Comment:
Looks like the previous and next errors in this same function use DH_PERROR_DER, instead of DH_PERROR_SYS. Should we be consistent?

Reply:
I agree we shouldn't update them all on this PR, but it was consistently DH_PERROR_DER before. Now this PR changed just one line to DH_PERROR_SYS. So it introduced inconsistency